src/pages/Dashboard.vue

```diff
@@ -80,11 +80,14 @@ export default defineComponent({
     const raceResult = util.promiseTimeout(7000, this.client.connectPublicApi())
     raceResult.then(async() => {
       if (this.client.isFirstLoad() === false) {
+        console.log("will start node now ***********")
```

Comment:
In IntelliJ you can commit part of the changes. No need to commit these debugging stuff.
You can also create change sets, just move debugging stuff into inactive change set and it will not be selected for committing in the first place.

Reply:
Thanks for the suggestion!